### DISCUSSION TITLE
Meteor shower issue when UTC hits 2025

### DISCUSSION THREAD

**Discussion by TheSuggestionGuy** [Stellarium/stellarium#3615] [General]
So, go to, somewhere in Africa, like Nouakchott (it's in UTC time so you can understand, this applies to every location) and remove the ground, and click on the Quadrantids, then go to the second before new year's 2025, then, you can see the meteor showers color is orange, and it has info saying: ZHRmax, Current ZHR, and Local Hourly Rate. Then on midnight of 2025, the meteor shower color becomes white, then Current ZHR, and Local Hourly Rate disappears, and ZHRmax becomes ZHRmax variable, and it's instead in a range. and it applies to all meteor showers, in all locations. so please fix this.

**Comment by gzotti** [comment]
If it's an issue, please report as issue.

**Reply by alex-w** [reply]
Please see bug report #3616
